### PR TITLE
chore: update swagger doc api endpoint

### PR DESCRIPTION
**Pull request by RiceAndMeet**

### CODE REVIEW

File: src/data/resources.json [hunk 1]
```diff
@@ -60,15 +60,15 @@
...
         },
         {
           "title": "Swagger 2",
-          "value": "https://lcd-axelar.stakerun.com/static/swagger/"
+          "value": "https://axelar-lcd.quickapi.com/static/swagger/"
         },
         {
           "title": "OpenAPI 1",
-          "value": "https://lcd-axelar.imperator.co/static/openapi/"
+          "value": "https://axelar-lcd.quickapi.com/static/openapi/"
```
Comment (milapsheth) [inline src/data/resources.json:71]: the next url is from stakerune which is also not supported

File: src/data/resources.json [hunk 2]
```diff
@@ -60,15 +60,15 @@
       "value": [
         {
           "title": "Swagger 1",
-          "value": "https://lcd-axelar.imperator.co/static/swagger/"
```
[the remark this answers is not in the document]
Comment (RiceAndMeet) [reply src/data/resources.json:63]: The issue with imperator LCD endpoint was an url routing issue at LB level, removing swagger will correctly route to swagger endpoint .

[the remark this answers is not in the document]
Comment (RiceAndMeet) [reply src/data/resources.json:63]: Imperator have adjust the url path to https://lcd-axelar.imperator.co/static/swagger

File: src/data/resources.json (hunk 1 above)
Comment (RiceAndMeet) [reply src/data/resources.json:71]: replace it with imperator endpoint

File: src/data/resources.json [hunk 3]
```diff
@@ -60,19 +60,19 @@
...
         },
         {
           "title": "OpenAPI 1",
-          "value": "https://lcd-axelar.imperator.co/static/openapi/"
+          "value": "https://axelar-lcd.quickapi.com/static/openapi/"
         },
         {
           "title": "OpenAPI 2",
-          "value": "https://lcd-axelar.stakerun.com/static/openapi/"
+          "value": "https://lcd-axelar.imperator.co/static/openapi/"
```
Comment (milapsheth) [inline src/data/resources.json:75]: make this the first one for consistent ordering?

**Review by milapsheth — APPROVED** [review]

File: src/data/resources.json [hunk 4]
```diff
@@ -60,19 +60,19 @@
       "value": [
         {
           "title": "Swagger 1",
-          "value": "https://lcd-axelar.imperator.co/static/swagger/"
+          "value": "https://lcd-axelar.imperator.co/static/swagger"
```
Comment (milapsheth) [inline src/data/resources.json:63]:
```suggestion
          "value": "https://lcd-axelar.imperator.co/static/swagger/"
```

File: src/data/resources.json [hunk 5]
```diff
@@ -60,19 +60,19 @@
...
         },
         {
           "title": "Swagger 2",
-          "value": "https://lcd-axelar.stakerun.com/static/swagger/"
+          "value": "https://axelar-lcd.quickapi.com/static/swagger"
```
Comment (milapsheth) [inline src/data/resources.json:67]:
```suggestion
          "value": "https://axelar-lcd.quickapi.com/static/swagger/"
```

File: src/data/resources.json [hunk 6]
```diff
@@ -60,19 +60,19 @@
...
         },
         {
           "title": "OpenAPI 1",
-          "value": "https://lcd-axelar.imperator.co/static/openapi/"
+          "value": "https://lcd-axelar.imperator.co/static/openapi"
```
Comment (milapsheth) [inline src/data/resources.json:71]:
```suggestion
          "value": "https://lcd-axelar.imperator.co/static/openapi/"
```

File: src/data/resources.json [hunk 7]
```diff
@@ -60,19 +60,19 @@
...
         },
         {
           "title": "OpenAPI 2",
-          "value": "https://lcd-axelar.stakerun.com/static/openapi/"
+          "value": "https://axelar-lcd.quickapi.com/static/openapi"
```
Comment (milapsheth) [inline src/data/resources.json:75]:
```suggestion
          "value": "https://axelar-lcd.quickapi.com/static/openapi/"
```

**Review by milapsheth — APPROVED** [review]

**Pull request merged**